index.html

```diff
@@ -1917,7 +1917,12 @@ <h3>Presentations</h3>
 <a>verifiable credentials</a> is a typical use of
 <a>verifiable presentations</a>.
         </p>
-
+        <p>
+<a>Verifiable presentations</a> are intended to be extremely short-lived, and
```

Comment:
This does not seem to align with the normative requirements we have in the spec.

Reply:
I disagree - see my [post](https://github.com/w3c/vc-data-model/issues/937#issuecomment-1265409715). Also, no example has been provided why VPs are intended to be (extremely) short lived, as suggested by @Sakurann s [comment](https://github.com/w3c/vc-data-model/issues/937#issuecomment-1387552819). So, lacking any substantiation for the assertion that presentations are intended to be extremely short-lived, and noting that the standard cannot know the intentions of holders regarding the presentations they create,I suggest to not merge this.